docs/semgrep-app/scm.md

```diff
@@ -0,0 +1,65 @@
+---
+slug: scm 
+append_help_link: true
+title: Integrating Semgrep into source code management (SCM) tools
+description: "Integrate Semgrep into self-hosted and custom SCM tools such as GitHub Enterprise and GitLab Self Hosted."
+hide_title: true
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Integrating Semgrep into source code management (SCM) tools
+
+Semgrep App's Team tier supports repositories hosted on both **GitHub Enterprise (GHE)** and **GitLab Self Managed (GLSM)** plans. Create multiple integrations in different self-hosted or custom servers.
+
+## Prerequisites
+
+You need the following permissions to integrate Semgrep into GHE or GLSM servers:
+
+* Permission to create a personal access token (PAT) for the repository to scan Semgrep on. This PAT is used to enable Semgrep App to create pull request (PR) or merge request (MR) comments for findings that can be [autofixed](../notifications/#automatically-fix-your-findings-through-pull-or-merge-requests).
+* Permission to add CI/CD secrets into your [GitHub](https://docs.github.com/en/actions/security-guides/encrypted-secrets) or [GitLab](https://docs.gitlab.com/ee/ci/secrets/) environments.
+
+## Integrating Semgrep into GitHub Enterprise or GitLab Self Managed 
+
+Integrate Semgrep into these custom source code management (SCM) tools by following the steps below:
+
+1. Sign in to [Semgrep App](https://semgrep.dev/login).
+2. Click **[Settings](https://semgrep.dev/orgs/-/settings)** > **Create new parent organization**.
+
+<div class = "bordered">
+
+![Screenshot of settings for parent organization](../img/app-parent-org.png "Screenshot of settings for parent organization")
+
+</div>
+
+3. Enter the name of your parent organization.

```

Comment:
Do we explain what a parent organization is and/or should we provide context/guideline on what the name should be here?

In the UI it now defaults to the name of their organization. Perhaps we could instead say something like:

> Confirm the name of your organization. If needed or preferred, you can enter a new name at this time.

Reply:
It actually doesn't let the user change the organization name. It displays a dialog with the deployment name as the new parent organization name.

Reply:
As a V1, I could see this just being a confirmation dialog that you confirm and that then logs you out :)

Reply:
Okay, I'll update the text :)